src/kapil/Switch/BankSwitch.java

```diff
@@ -0,0 +1,165 @@
+	    	   else
+	    		   System.out.println("Balance is not sufficient ");
+	    	   break;
+		   case "hdfc" :
+	    	   if(hdfcBalance>= amt) {
+	    		   hdfcBalance = hdfcBalance - amt;
+	    	   hdfcDebitCount = hdfcDebitCount + 1;	
+			   totalDebitCount = totalDebitCount + 1;
+			   System.out.println("hdfc debit amount is "+hdfcBalance);
+		   }else
+	    		   System.out.println("Balance is not sufficient ");
+	    	   break;}
+		   break;
+	   }   	    	   
+		   
+		  
+	   } 
+	   void operation2(String bank, String op, int amount) {
+	   switch(op) {
+	     case"Credit" :
+		   switch (bank) {
+		   case "icici": 
+		      if(amount <= iciciBalance ) {
+			     iciciBalance =iciciBalance + amount;
+			     iciciCreditCount = iciciCreditCount + 1;
+			     totalCreditCount = totalCreditCount + 1 ;
+		         System.out.println("Credit amount is "+iciciBalance);}
+		         else
+		        	 System.out.println("Balance is not Sufficient");
+		          break;
+		   
+		 case "bob": 
+			   if(amount <= bobBalance ) {
+				     bobBalance =bobBalance + amount;
+				     bobCreditCount = bobCreditCount + 1;
+				     totalCreditCount = totalCreditCount + 1 ;
+			         System.out.println("Credit amount is "+bobBalance);}
+			      else
+			        	 System.out.println("Balance is not Sufficient");
+			          break;
+	     case "sbi": 
+	    	 if(amount <= sbiBalance ) {
+				     sbiBalance = sbiBalance + amount;
+				     sbiCreditCount = sbiCreditCount + 1;
+				     totalCreditCount = totalCreditCount + 1 ;
+				         System.out.println("Credit amount is "+sbiBalance);}
+				      else
+				    	  System.out.println("Balance is not Sufficient");
+				          break;
+		case "hdfc": 
+			  if(amount <= hdfcBalance ) {
+			     hdfcBalance =hdfcBalance + amount;
+			    		 hdfcCreditCount = hdfcCreditCount + 1;
+						  totalCreditCount = totalCreditCount + 1 ;
+					         System.out.println("Credit amount is "+hdfcBalance);}
+					      else
+					         System.out.println("Balance is not Sufficient");
+					       break;}   
+	   
+	   }
+	    
+	   }
+	   void totalbal() {
+		   System.out.println("Total balance is " +(iciciBalance+ bobBalance+sbiBalance+hdfcBalance));
+	   }
+	   void debitCreditoperation() {
+		   System.out.println("total debit operation " +totalDebitCount);
+		   System.out.println("total credit operation "+totalCreditCount);
+	   }
+	   void individualDebit() {
+		   System.out.println("icici debit count :" +iciciDebitCount);
+		   
+	   }
+	   
+	   void individualCredit() {
+		   System.out.println("icici credit count "+iciciCreditCount);
+		   System.out.println("bob credit count "+bobCreditCount);
+	   }
+	   
+	public static void main(String[] Args) {
+		BankSwitch bs = new BankSwitch();
+		bs.operation1("icici","Debit", 5000);
+		bs.operation1("icici","Debit", 3000);
+		bs.operation1("bob","Debit", 6000);
+		bs.operation1("sbi","Debit", 7000);
```

Comment:
You don't require two different method for credit and debit, you can handle this in single method as well

take reference of below code
Vaibhav. - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/600/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/598/files